tools/release/ltsgraph/utility.h

```diff
@@ -67,7 +68,7 @@ constexpr float PI_2 = PI * 0.5f;
 inline
 float frand(float min, float max)
 {
-  return (static_cast<float>(qrand()) / RAND_MAX) * (max - min) + min;
+  return static_cast<float>(QRandomGenerator::global()->generateDouble() * (max - min) + min);
```

Comment:
It appears to me that random generators are not a Qt matter. 
Wouldn't it be more proper to use a random generator provided
by the C++ library, for instance from: #include <random>.

Reply:
Originally the Qt function `qrand` was used. Since the function is used in Qt worker threads the rationale for this is probably thread safety. The standard C++11 random generators are not thread safe.